en/qgc/video_streaming_wifi_broadcast.md

```diff
@@ -1,23 +1,26 @@
-# Long-distance Video Streaming in QGroundControl
+# Long-distance Video Streaming and telemetry via raw WiFi radio
 
-This page shows how to set up a companion computer with a camera (Logitech C920 or RaspberryPi camera) such that the video stream is transferred from the UAV to a ground computer and displayed in *QGroundControl*. This setup uses WiFi in unconnected (broadcast) mode and software from the [Wifibroadcast project](https://github.com/svpcom/wifibroadcast/wiki).
+This page shows how to set up a companion computer with a camera (Logitech C920 or RaspberryPi camera) such that the video stream is transferred from the UAV to a ground computer and displayed in *QGroundControl*. Also it provides support for bidirectional telemetry like SiK radio does. This setup uses WiFi in unconnected (broadcast) mode and software from the [Wifibroadcast project](https://github.com/svpcom/wifibroadcast/wiki).
```

Comment:
Perhaps "Also it provides support for bidirectional telemetry like SiK radio does. " TO "The mechanism also provide a bidirectional telemetry link (i.e. like SiK radio). "